e2e/utils/mailhog.ts

```diff
@@ -0,0 +1,30 @@
+type Email = {
+  Content: { Headers: { Subject: string[] } };
+};
+
+type AllEmails = {
+  items: Email[];
+};
+
+const host = process.env.MAILHOG_HOST || 'localhost';
+
+export const getAllEmails = async (): Promise<AllEmails> => {
+  const res = await fetch(`http://${host}:8025/api/v2/messages`);
+  return res.json() as Promise<AllEmails>;
+};
+
+export const getFirstEmail = (allEmails: { items: Email[] }) => {
+  return allEmails.items[0];
+};
+
+export const getSubject = (email: {
+  Content: { Headers: { Subject: string[] } };
+}) => {
+  return email.Content.Headers.Subject[0];
+};
+
+export const deleteAllEmails = async () => {
+  await fetch(`http://${host}:8025/api/v1/messages`, {
```

Comment:
Why are we using two different API versions?

Reply:
Weirdly delete doesn't exist in V2. At least not according to their spec, I didn't actually check.

Reply:
Okay, I've checked. DELETE /api/v1/messages works, but DELETE /api/v2/messages is a 404 :shrug:

Reply:
😭